lasagne/tests/layers/test_pool.py

```diff
@@ -126,7 +126,8 @@ def pool_test_sets_ignoreborder():
                     yield (pool_size, stride, pad)
 
     def input_layer(self, output_shape):
-        return Mock(get_output_shape=lambda: output_shape)
+        return Mock(get_output_shape=lambda: output_shape,
+                    output_shape=output_shape)
```

Comment:
`get_output_shape` can be omitted, that's deprecated anyway. Would be nice if you could just fix that. I wonder why the tests still worked, though? Did we overlook something in #182?